app/assets/stylesheets/styles/ui/inputs/inputs.scss

```diff
@@ -19,3 +19,7 @@ textarea {
 input.ng-invalid {
   border: 1px solid $alert-color;
 }
+
+input#search_crop_name.ng-invalid {
```

Comment:
@Katy310. I think this is a fair comment. I know that I suggested the above selector, but maybe there's a better way to do so? Do you have time to make a fix to your PR?